submissions/josephOlasimbo/database/init.sql

```diff
@@ -0,0 +1,48 @@
+-- CREATE THE food bag DB
+CREATE DATABASE food_bag;
+USE food_bag;
+
+-- CREATE THE food TABLE
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY (id)
```

Comment:
this table shoud contain type too with the datatype as ENUM NOT NULL with the four type food categories